openml-datarobot/README.md

```diff
@@ -1,12 +1,12 @@
 # DataRobot API
 
-This module contains the classes and logic related with the import of a model generated in [DataRobot](https://www.datarobot.com/).
+This module contains the classes and logic that allow to import models generated in [DataRobot](https://www.datarobot.com/).
 
-The current implementation allows to import a binary classification model exported in a Jar file.
+You can use the current implementation to import a binary classification model exported in a Jar file.
```

Comment:
I think this has to be somewhat in bold: that the model must be exported as Jar file from DataRobot API / UI.